lms/services/email_preferences.py

```diff
@@ -16,12 +17,25 @@ class InvalidTokenError(Exception):
     pass
 
 
+KEY_PREFIX = "instructor_email_digests.days."
+
+
 class EmailPreferencesService:
-    def __init__(self, db, jwt_service: JWTService, secret: str, route_url: Callable):
+    DAY_KEYS = [f"{KEY_PREFIX}{day}" for day in [1, 2, 3, 4, 5, 6, 7]]
```

Comment:
This is part of the class because the forthcoming views are going to need to read `email_preferences_service.DAY_KEYS`. They don't need to read `KEY_PREFIX`. Also, if `KEY_PREFIX` were in the class then `DAY_KEYS` wouldn't be able to reference it like this!